examples/MultiAxisExample.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Add path to library (just for examples; you do not need this)
+import initExample
+import numpy as np
+import pyqtgraph as pg
+from pyqtgraph.Qt import QtCore, QtGui
+
+app = QtGui.QApplication([])
+mw = QtGui.QMainWindow()
+mw.resize(800, 800)
+pg.setConfigOption("background", "w")
+pg.setConfigOption("foreground", "k")
+
+mpw = pg.MultiAxisPlotWidget()
+
+mw.setCentralWidget(mpw)
+mw.show()
+
+# TITLE
+mpw.setTitle("MultiAxisPlotWidget Example")
+# AXYS
+mpw.addAxis("samples1", "bottom", "Samples1", "samples1")
+mpw.addAxis("samples2", "bottom", "Samples2", "samples2")
+mpw.addAxis("sin1", "left", "Data1", "sin1")
+mpw.addAxis("sin2", "left", "Data2", "sin2")
+# CHARTS
+mpw.addChart("Dataset 1", "sin1", "samples1")
+mpw.addChart("Dataset 2", "sin1", "samples2")
+mpw.addChart("Dataset 3", "sin2", "samples2")
+# make and display chart
+mpw.makeLayout()
+# LEGEND
+mpw.addLegend(offset=(0, 0))
```

Comment:
I don't see a legend when I run this, do you know what that's about?

Reply:
legend must be added before the creation of the charts, fixed the example now